be/src/storage/lake/lake_persistent_index.cpp

```diff
@@ -445,4 +446,12 @@ Status LakePersistentIndex::load_from_lake_tablet(TabletManager* tablet_mgr, con
     return Status::OK();
 }
 
+size_t LakePersistentIndex::memory_usage() const {
+    size_t mem_usage = _memtable->memory_usage();
```

Comment:
Will `_memtable` be nullptr?

Reply:
It will be constructed in constructor of LakePersistentIndex.

Reply:
It's best not to make optimistic assumptions